qiskit_experiments/framework/composite/composite_analysis.py

```diff
@@ -99,7 +99,11 @@ def _run_analysis(self, experiment_data: ExperimentData, **options):
             # Run analysis
             # Since copy for replace result is handled at the parent level
             # we always run with replace result on component analysis
-            sub_exp.run_analysis(sub_exp_data, replace_results=True)
+            experiment_data.add_analysis_callback(
+                self._component_callback,
+                index=component_ids[i],
+                component_analysis=sub_exp.run_analysis,
+            )
```

Comment:
Seems like this is sequential execution. Is there any option that allows to skip blocking?

Reply:
Is this background process? Is so this doesn't matter